apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -615,13 +615,18 @@ listeners([]) ->
                     {error, _} -> [];
                     MC -> [{max_conns, MC}]
                 end,
+            ShutdownCount =
+                case emqx_listeners:shutdown_count(ID, Bind) of
+                    {error, _} -> [];
```

Comment:
Since the error message is to be discarded, is it necessary to return an error tuple